components/cookie/banner/src/index.js

```diff
@@ -87,11 +87,9 @@ class CookieBanner extends Component {
 
   render () {
     const { showCookieBanner } = this.state
-    const hasAcceptedCookies = this._getHasAcceptedCookie()
-    if (hasAcceptedCookies || !showCookieBanner) {
+    if (!showCookieBanner || this._getHasAcceptedCookie()) {
```

Comment:
When server renders, it won't execute `_getHasAcceptedCookie()` method that tries to read `document` var.